js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java

```diff
@@ -76,10 +85,30 @@ public void loadModel(String uri, ReadableMap options, Promise promise) {
     }
   }
 
+  /**
+   * React native binding API to load a model using the BASE64 encoded model data.
+   *
+   * @param data the BASE64 encoded model data.
+   * @param options onnxruntime session options
+   * @param promise output returning back to react native js
+   * @note the value provided to `promise` includes a key representing the session.
+   *       when run() is called, the key must be passed into the first parameter.
+   */
+  @ReactMethod
+  public void loadModelFromBase64EncodedBuffer(String data, ReadableMap options, Promise promise) {
+    try {
+      byte[] modelData = Base64.decode(data, Base64.DEFAULT);
+      WritableMap resultMap = loadModel(modelData, options);
+      promise.resolve(resultMap);
+    } catch (Exception e) {
+      promise.reject("Can't load model from buffer: " + e.getMessage(), e);
```

Comment:
```suggestion
      promise.reject("Failed load model from buffer: " + e.getMessage(), e);
```
nit: "can't" makes it sound like we don't support loading from buffer.